src/common/services/api/order.service.js

```diff
@@ -253,6 +253,7 @@ class Order {
         const postData = cvv ? { 'security-code': cvv } : {}
         postData['cover-cc-fees'] = !!this.retrieveCoverFeeDecision()
         postData['radio-call-letters'] = this.retrieveRadioStationCallLetters()
+        postData['premium-code'] = this.retrievePremiumCode()
```

Comment:
This isn't here, it is at the same place `campaign-code` is at.

Reply:
I saw that `campaign-code` is added to `itemConfig` in this file: https://github.com/CruGlobal/give-web/blob/master/src/app/branded/step-1/branded-checkout-step-1.component.js#L34
Should I add `premium-code` to itemConfig instead?  I was trying to follow the app flow to see what happens to the properties in `itemConfig`, I can't see where this data gets submitted to the API.

Reply:
Yes, `itemConfig` is the place to put it.